src/se/content.rs

```diff
@@ -1410,19 +1367,20 @@ pub(super) mod tests {
                 => Unsupported("character `<` is not allowed at the start of an XML name `<\"&'>`"));
 
             value!(newtype: Newtype(42) => "42");
-            value!(enum_newtype: Enum::Newtype(42) => "<Newtype>42</Newtype>");
+            value!(enum_newtype: Enum::Newtype(42) => "\n  <Newtype>42</Newtype>\n  ");
 
             // Note that sequences of primitives serialized without delimiters!
-            value!(seq: vec![1, 2, 3] => "1\n  2\n  3");
-            value!(seq_empty: Vec::<usize>::new());
+            value!(seq: vec![1, 2, 3] => "123");
+            value!(seq_empty: Vec::<usize>::new() => "");
             value!(tuple: ("<\"&'>", "with\t\n\r spaces", 3usize)
-                => "&lt;&quot;&amp;&apos;&gt;\n  \
-                    with\t\n\r spaces\n  \
+                => "&lt;&quot;&amp;&apos;&gt;\
+                    with\t\n\r spaces\
                     3");
-            value!(tuple_struct: Tuple("first", 42) => "first\n  42");
+            value!(tuple_struct: Tuple("first", 42) => "first42");
```

Comment:
I'm definitely missing something here.

How would `Tuple("first42", 0)` be distinguished from `Tuple("first", 420)`

Reply:
The same answer as above. This is restriction of current implementation. I have a TODO to forbid serialization consequent text fields:
https://github.com/tafia/quick-xml/blob/6eea6bbcb361a3ab51ccd1f5917d855d8a195c71/src/se/content.rs#L75

Reply:
Is there an issue written up for that already, and if not can you write one and reference it

Reply:
No, there is no issue yet

Reply:
In #823 I changed the return value in that case to error